debian/autobake-deb.sh

```diff
@@ -22,9 +22,9 @@ if [[ -d storage/columnstore/columnstore/debian ]]; then
   cp -v storage/columnstore/columnstore/debian/mariadb-plugin-columnstore.* debian/
   echo >> debian/control
   cat storage/columnstore/columnstore/debian/control >> debian/control
-  # Don't build ColumnStore as part of the native build, only build it when triggered
-  # by autobake-deb.sh
-  sed 's|#CMAKEFLAGS += -DPLUGIN_COLUMNSTORE=YES|CMAKEFLAGS += -DPLUGIN_COLUMNSTORE=YES|' -i debian/rules
+  # ColumnStore is explcitly disabled in the native build, so allow it now
+  # when build it when triggered by autobake-deb.sh
+  sed '/-DPLUGIN_COLUMNSTORE=NO/d' -i debian/rules
```

Comment:
`s/-DPLUGIN_COLUMNSTORE=NO//` might be a bit more future-proof. In case someone adds another cmake option later on the same line. There's no rule one option per line there.

Reply:
I had that at first but was thinking this is more elegant and does not leave empty lines behind..

Reply:
Without this both https://salsa.debian.org/mariadb-team/mariadb-server/-/pipelines/210408 and Travis-CI would fail. ColumnStore makes the build time 2-3x longer and is impossible to get passing on Travis-CI or Salsa-CI.